how-to-guides/consensus-node.md

```diff
@@ -219,10 +219,16 @@ a trusted RPC endpoint (/block). RPC endpoints are also required for retrieving
 light blocks. These can be found in the docs here under the respective networks or
 from [the chain-registry](https://github.com/cosmos/chain-registry).
 
-In `$HOME/.celestia-app/config/config.toml`, set
+::: warning
+State sync requires at least two RPC endpoints to function. These endpoints are used to:
+1. Query the trusted header (height and hash)
+2. Retrieve light blocks for verification
+:::
+
+In `$HOME/.celestia-app/config/config.toml`, set the RPC endpoints:
```

Comment:
I think we'd also need to leave a note here to set trusted height and hash, because it isn't clear to me. if you set height/hash higher than 1, for example at 5000, do you not sync anything before that height 5000?